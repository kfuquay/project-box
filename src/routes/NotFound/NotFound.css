.not-found {
    text-align: center;
    font-size: 26px;
    color: whitesmoke;
    margin: 20vh 0;
}