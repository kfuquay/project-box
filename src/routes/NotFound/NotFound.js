import React from 'react'
import './NotFound.css'

function NotFound() {
    return(
        <div>
            <p className="not-found">Page Not Found</p>
        </div>
    )
}

export default NotFound;