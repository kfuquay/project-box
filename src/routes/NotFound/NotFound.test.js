import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import NotFound from './NotFound'

describe(`NotFound component`, () => {
  it("renders without crashing", () => {
    const wrapper = shallow(<NotFound />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
