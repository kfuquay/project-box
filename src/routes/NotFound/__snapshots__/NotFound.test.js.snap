// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component renders without crashing 1`] = `
<div>
  <p>
    Page Not Found
  </p>
</div>
`;
